doc/reference/terminology.rst

```diff
@@ -122,32 +123,33 @@ pre-kernel contexts only when the parameter selects the no-wait path.
 isr-ok
 ======
 
-The isr-ok attribute is used on a function to indicate that it can be
-called from interrupt context.  If necessary the function will use
-:cpp:func:`k_is_in_isr` to detect its calling context and force an
-execution path that will not cause the invoking thread to sleep.
+The isr-ok attribute is used on a function to indicate that it works as
```

Comment:
I think this is an awkward wording that ends up not adding much value. "indicate that it works as documented....". Why do we need this if we have to rely on the documentation, IMO the original text was more to the point: functions with this attribute can be called from interrupt context, this was straight and to the point.

Reply:
ok